technic/spec/fixtures/technic.conf

```diff
@@ -11,19 +11,18 @@ enable_vacuum = true
 
 multimeter_remote_start_ttl = 300
 
-enable_wind_mill = true
-enable_frames = true
-enable_nuclear_reactor_digiline_selfdestruct = true
+enable_wind_mill = false
+enable_nuclear_reactor_digiline_selfdestruct = false
 quarry_dig_above_nodes = 3
 quarry_max_depth = 100
-quarry_time_limit = 5000
+quarry_time_limit = 3000
 
 switch_off_delay_seconds = 1800
 network_overload_reset_time = 20
 admin_priv = basic_privs
 enable_corium_griefing = true
 enable_radiation_protection = true
-enable_radiation_throttling = true
+enable_radiation_throttling = false
```

Comment:
If I remember correctly, I've disabled radiation throttling for tests simply because it is time dependent functionality which required predictable timing for tests.

Mineunit didn't have predictable timings back then, now it has this ability but to enable it for tests one should also make sure that all tests are compatible with functionality that requires predictable timings.

This is because if tests are not aware of this or do not use predictable timing it can create situation where tests pass or fail randomly because of variation in host machine scheduling and performance.

Because of this I'd recommend not enabling this feature for tests unless you're also willing to write specific tests for it.

Reply:
All I said there is probably wrong because this PR is disabling it while it actually was enabled for tests.

Actually now _I think_ I remember that situation with radiation tests was related to timing but decision was to not include radiation tests at all. So basically while throttling is enabled it simply loads more features in but those features are not actually tested.

Likely best to just not touch it until actually working on related tests.